src/frontend-samples/thematic-display-sample/ThematicDisplayApp.tsx

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import * as React from "react";
+import { Range1dProps } from "@bentley/geometry-core";
+import { ThematicDisplay, ThematicDisplayProps, ThematicGradientColorScheme } from "@bentley/imodeljs-common";
+import { Viewport, ViewState3d } from "@bentley/imodeljs-frontend";
+import ThematicDisplaySampleUI from "./ThematicDisplayUI";
+
+// cSpell:ignore imodels
+
+/** This file contains the user interface and main logic that is specific to this sample. */
+export class ThematicDisplayAPI {
```

Comment:
This split between API and APP is interesting but we don't do it in the rest of the samples.  Seems better to follow the common pattern.